keras/config.py

```diff
@@ -0,0 +1,13 @@
+from __future__ import absolute_import, print_function
+
+import os
+
+KERAS_DIR = os.environ.get('KERAS_DIR', os.path.expanduser('~/.keras'))
```

Comment:
This should be defined in `backend/common`, and should be accessed publicly via a function. See other examples in `backend/common`.